lib/assets/javascripts/bootstrap_listeners.js

```diff
@@ -15,9 +15,10 @@ $(document).ready(function(){
         $(this).parent().find('.minus-laranja').removeClass('minus-laranja').addClass('plus-laranja');
     });
     // Initialises all tooltips present on a page
-    $('.has-tooltip').tooltip({
-        placement: "right",
-        trigger: "focus"
-    });
+    $('.has-tooltip').tooltip({ placement: "right", trigger: "focus" });
     $(".help").popover();
+    //Initiliases all popovers on a page
```

Comment:
I noticed today when refactoring admin.js that there were multiple classes to trigger the same behaviour. That's the example provided in bootstrap website. Your example is good too but using data-* does not tight to a specific html tag.